library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1255,7 +1270,25 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
           mimeType = MimeTypes.AUDIO_FLAC;
           initializationData = Collections.singletonList(codecPrivate);
           break;
-        case CODEC_ID_SUBRIP:
+        case CODEC_ID_ACM:
+
+          WAVEFORMATEX struct = parseMsAcmCodecPrivate(new ParsableByteArray(codecPrivate));
+          if (struct.wFormatTag == ACM_FORMATTAG_A_PCM_INT_LIT ||
+                  (struct.wFormatTag == WAVEFORMATEXTENSIBLE.WAVE_FORMAT_EXTENSIBLE
+                  && (struct instanceof WAVEFORMATEXTENSIBLE) && Arrays.equals(((WAVEFORMATEXTENSIBLE)struct).SubFormat, A_MS_ACM_GUID_PCM))) {
+
+              mimeType = MimeTypes.AUDIO_RAW;
+              if (audioBitDepth != 16)
+                  throw new ParserException("ExoPlayer is only capable of handling 16-bit audio.");
```

Comment:
As a side note, we're working on adding support for re-sampling of 8/24-bit audio to. This will likely happen all the way down in AudioTrack. Or in other words, don't bother working on this, since it's near the top of our list already ;).